Bottom: a820607d2ceadb148d2eb04c532fc14de4e8383a
Top:    afdfe5926f79ac24e0601c1df583d506c6aed24c
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2016-02-24 20:18:56 +0000

Disable polygon heritage


---

diff --git a/src/polygon1.c b/src/polygon1.c
index 950ea0d..885f268 100644
--- a/src/polygon1.c
+++ b/src/polygon1.c
@@ -2848,8 +2848,10 @@ poly_Boolean_free (POLYAREA * ai, POLYAREA * bi, POLYAREA ** res, int action)
   *res = NULL;
 
   /* Make copies for tracking polygon parentage (DEBUG) */
-  if (!poly_M_Copy0 (&a_copy, a) || !poly_M_Copy0 (&b_copy, b))
-      return err_no_memory;
+//  if (!poly_M_Copy0 (&a_copy, a) || !poly_M_Copy0 (&b_copy, b))
+//      return err_no_memory;
+  a_copy = NULL;
+  b_copy = NULL;
 
   /* Move the parentage information over onto the copy */
   if (a_copy != NULL)
